### ISSUE TITLE
Lot of redirects rules on Netlify deploys

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#12022]
While working on the redirects, I noticed that the project is currently generating a lot of redirect rules automatically, ~4k. Debugging it, most of these redirects (~3.5k) are coming from the files that live inside the /public folder. For each file, Next creates a redirect rule.

This brings up the discussion of whether we need to put .md files in the public folder, since we only need them at build time. Or even if we need to place images or assets that are only consumed at build time.

![image](https://github.com/ethereum/ethereum-org-website/assets/468158/dcbede4f-25be-48c5-b2f2-6e56ed905e0b)

**Comment by TylerAPfledderer** [comment]
@pettinarip can view [this netlify blog tutorial](https://www.netlify.com/blog/2020/05/04/building-a-markdown-blog-with-next-9.4-and-netlify/) for possible clues on moving the files someplace else in the directory. It seems like to can put them almost anywhere. Don't think Crowdin should be impacted aside from updating configurations.

**Comment by pettinarip** [comment]
@TylerAPfledderer correct, we don't need the md files in the public folder. We could move those easily.

Another question has to do with the images. Are we going to have problems with the images that live next to each md file if we also move them out of the public folder? probably we need to tweak some logic around them.

**Comment by TylerAPfledderer** [comment]
> @TylerAPfledderer correct, we don't need the md files in the public folder. We could move those easily.
> 
> Another question has to do with the images. Are we going to have problems with the images that live next to each md file if we also move them out of the public folder? probably we need to tweak some logic around them.

Hmm that shouldn't matter. But images as static assets are supposed to stay in the `public` folder.

**Comment by wackerow** [comment]
@pettinarip Any update here?

**Comment by pettinarip** [comment]
@wackerow I think we will be able to simplify this with the latest upgrades of Nextjs and the Netlify runtime plugin. We should use the redirects mechanism from Next instead of Netlify. Something to test. I'll try to do this once we finish with the performance improvements/fixes.

**Comment by wackerow** [comment]
Okay great; assigned you here, feel free to update/close out as needed

**Comment by corwintines** [comment]
Not noticing this issue in the deploy summary anymore, so going to close this out.
